presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -172,11 +172,19 @@ Unit              Description
 .. function:: date_add(unit, value, timestamp) -> [same as input]
 
     Adds an interval ``value`` of type ``unit`` to ``timestamp``.
-    Subtraction can be performed by using a negative value.
+    Subtraction can be performed by using a negative value::
+
+        SELECT date_add('second', 86400, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-03-02 00:00:00.000 UTC
```

Comment:
would be good to add small second value (eg 86), to show how seconds/minutes get updated. -- like you did with `hour` example below